odl/contrib/datasets/ct/examples/mayo_reconstruct.py

```diff
@@ -1,29 +1,38 @@
 """Reconstruct Mayo dataset using FBP and compare to reference recon.
 
 Note that this example requires that Mayo has been previously downloaded and is
-stored in the location indicated by "mayo_dir".
+stored in the location indicated by "proj_folder" and "rec_folder".
 
 In this example we only use a subset of the data for performance reasons,
-there are ~48 000 projections in the full dataset.
+there are ~32 000 projections per patient in the full dataset.
 """
-
+import numpy as np
 import odl
 from odl.contrib.datasets.ct import mayo
+from time import perf_counter
 
-mayo_dir = ''  # replace with your local folder
+# define data folders
+proj_folder = odl.__path__[0] + '/../../data/LDCT-and-Projection-data/' \
+             'L004/08-21-2018-10971/1.000000-Full dose projections-24362/'
+rec_folder = odl.__path__[0] + '/../../data/LDCT-and-Projection-data/' \
+             'L004/08-21-2018-84608/1.000000-Full dose images-59704/'
 
-# Load reference reconstruction
-volume_folder = mayo_dir + '/Training Cases/L067/full_1mm_sharp'
-partition, volume = mayo.load_reconstruction(volume_folder)
+# Load projection data
+print("Loading projection data from {:s}".format(proj_folder))
+geometry, proj_data = mayo.load_projections(proj_folder,
+                                            indices=slice(16000, 19000))
```

Comment:
Same question as above: why change the slice?

Reply:
It has been adjusted to the example considered (patient L004). In this case, the projection data counts ~32k images and the above slice choice reconstructs a central part of the volume.